index.html

```diff
@@ -148,15 +197,11 @@ <h2>Get started</h2>
       <article>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
         dolore magna.</article>
       <form action="https://formspree.io/f/mjvdryev" method="POST">
-        <div class="the-relative">
-          <input type="text" maxlength="30" placeholder="Your name" required>
-        </div>
-        <div class="the-relative">
-          <input type="email" placeholder="Your email" required>
-        </div>
-        <div class="the-relative">
-          <input type="text" maxlength="500" class="text-area" placeholder="Enter your message here.." required>
-        </div>
+        <ul class="the-relative">
+         <li class="tname"> <h3>Your name</h3><input type="text" maxlength="30" placeholder="Your name" required></li> 
+         <li class="mail"><h3>Your email</h3><input type="email" placeholder="Your email" required></li> 
+         <li class="texts"><h3>Message</h3><input type="text" maxlength="500" class="text-area" placeholder="Enter your message here.." required></li> 
+        </ul>
         <button class="st_colla">Start collaboration</button>
       </form>
     </div>
```

Comment:
- [ ] [OPTIONAL] This is for the footer section. Implementing according to Figma by adding social media links (icons) may really change the look of your page, just like this:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/222931079-61f5ec20-527f-41cc-b276-71a02bbedb1e.png)|![image](https://user-images.githubusercontent.com/99715640/222931098-d4038598-1098-4194-b385-0742ae4afb3a.png)|

- [ ] [OPTIONAL] There's an horizontal scrollbar on your portfolio page which is not really good for user experience. Kindly consider inspecting for the element causing this overflow so that you can stabilize the flow of your page on several devices :wink: